src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Test
+{
+    [Trait("category", "outerloop")]
```

Comment:
First thing I tried!
`"Attribute 'OuterLoop' is not valid on this declaration type.  It is only valid on 'method' declarations"`.
Debated about reporting/fixing it myself, but wasn't sure it was a deliberate choice or not.